packages/block-library/src/image/image.js

```diff
@@ -433,6 +433,9 @@ export default function Image( {
 						isResizable={ isResizable }
 						imageWidth={ naturalWidth }
 						imageHeight={ naturalHeight }
+						imageSizeHelp={ __(
+							"Select which image size to load. This changes the resolution of the image, not it's visual size."
```

Comment:
There's a small typo in the proposed description it should be [`its` instead of `it's`](https://www.grammarly.com/blog/its-vs-its/).

Reply:
Thanks!